terraform/main.tf

```diff
@@ -132,6 +132,12 @@ data "aws_ecr_image" "report-image" {
   image_tag       = "latest"
 }
 
+data "aws_ecr_image" "transform-image" {
+  repository_name = "c13-lakshmibai-transform-load"
```

Comment:
It might be a good idea to keep this repo name as an env variable, given that we're doing the same in the individual scripts.